database-mysql/index.js

```diff
@@ -1 +1,33 @@
 // this is where the database connection will happen 
+const Sequelize = require('sequelize');
+
+const host = 'localhost';
+const user = 'root';
+const port = 3306;
+const password = '';
+const database = 'joke-me-off';
+
+const connection = new Sequelize(database, user, password, {
+  dialect: 'mysql',
+  host,
+  port,
+});
+
+const Users = connection.define('users', {
+    name: Sequelize.STRING,
+    user_name: Sequelize.STRING,
```

Comment:
since we're using sequelize which lets use work with javascript and js objects, lets use camel case for variable names
user_name to userName